app/payment-link-v2/confirm/confirm.njk

```diff
@@ -44,6 +44,8 @@
           }
         %}
 
+        {% set actionSettings = null %}
```

Comment:
Would it be good to have the summary elements as a macro or a sub-template so we don't have to worry about unsetting things?

Reply:
Good idea - I will separate out that code and create some Cypress tests.

Reply:
This has been separated out